Makefile

```diff
@@ -51,7 +51,7 @@ endif
 
 REGISTRY ?= gcr.io/k8s-staging-ingress-nginx
 
-BASE_IMAGE ?= k8s.gcr.io/ingress-nginx/nginx:v20210115-gba0502603@sha256:224da667cf3047998ea691e9766fedd1eab94257a39df81374bfa14536da3688
+BASE_IMAGE ?= timmysilv/nginx:build-yaml@sha256:75ff82ccbad0633643202044c85fbb9ab9211f18f7f0170b59a4edfaa573f38c
```

Comment:
based on what version of code was this base image built?

Reply:
The (now fixed - sorry) link from the previous comment.